pages/public_cloud/data_analytics/analytics/information_05_capabilities/guide.en-gb.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Data Analytics - Capabilities and Limitations
+excerpt: Discover the capabilities and limitations of Data Analytics
+updated: 2025-01-07
+---
+
+## Data Analytics
+
+With our data analytics services, we take care of the database infrastructure and administration, including set-up, backup, scalability, and security.
+Choose from a large portfolio of popular engines.
+
+We continuously improve our offers. You can follow and submit ideas to add to our roadmap at <https://github.com/orgs/ovh/projects/16/views/5>.
+
+## Database Engines
+
+We currently offer 2 data analytics engines:
+
+### M3db
```

Comment:
M3DB is a Databases service not Analytics.

Analytics services are: Kafka, Kafka Connect, Kafka MirrorMaker, OpenSearch, Dashboards (Grafana)